subprojects/resources-http/src/integTest/groovy/org/gradle/internal/resource/transport/http/CookieHeaderTest.groovy

```diff
@@ -37,7 +37,7 @@ class CookieHeaderTest extends Specification {
     @Rule
     ConfigureLogging logging = new ConfigureLogging(listener, LogLevel.WARN)
     SslContextFactory sslContextFactory = new DefaultSslContextFactory()
-    HttpSettings settings = new DefaultHttpSettings([], sslContextFactory)
+    HttpSettings settings = DefaultHttpSettings.builder().withAuthenticationSettings([]).withSslContextFactory(sslContextFactory).build()
```

Comment:
Would be nice to split the builder calls to their own lines.

Reply:
done.